package.json

```diff
@@ -74,30 +74,30 @@
     "portfinder": "1.0.20",
     "prettyjson": "1.2.1",
     "read-last-lines": "1.6.0",
-    "replace-in-file": "3.4.2",
+    "replace-in-file": "3.4.3",
     "rxjs": "5.5.11",
     "semver": "5.6.0",
     "shasum": "1.0.2",
     "stat-mode": "0.2.2",
     "strip-ansi": "5.0.0",
     "symlink-or-copy": "1.2.0",
-    "systeminformation": "3.52.0",
+    "systeminformation": "3.54.0",
     "tail": "2.0.2",
     "validator": "7.2.0",
     "yargs": "12.0.5",
-    "yarn": "1.12.3"
+    "yarn": "1.13.0"
   },
   "devDependencies": {
     "chai": "4.2.0",
     "coveralls": "3.0.2",
-    "eslint": "5.10.0",
-    "eslint-plugin-ghost": "0.0.27",
+    "eslint": "5.12.0",
+    "eslint-plugin-ghost": "0.1.0",
     "has-ansi": "3.0.0",
     "mocha": "5.2.0",
     "nyc": "13.1.0",
     "proxyquire": "2.1.0",
     "sinon": "7.2.2",
-    "standard-version": "4.3.0",
+    "standard-version": "4.4.0",
```

Comment:
standard-version was kept at 4.3.0 for a reason - there was a change in 4.4.0 that keeps wanting to rewrite the entire change log.

Keeping it at 4.3 means we can pretty easily release without having to manually go and update the changelog.

Reply:
Rolling back!